packages/@lwc/features/README.md

```diff
@@ -0,0 +1,62 @@
+# Features
+
+[RFC](https://github.com/salesforce/lwc-rfcs/blob/master/text/0111-feature-flags.md)
+
+## Compile-time flags
+
+## Runtime flags
+
+Runtime flags are disabled by default and can be enabled or disabled via
+`globalThis.LWC_config.features`. Set the flag to `true` to enable and set the
```

Comment:
This paragraph is a little bit confusing, who should set that there? and when? why is that affecting the compilation?